src/faux-macros.lisp

```diff
@@ -33,6 +33,9 @@
 (define-coalton-editor-macro coalton:define-type (name &body definition)
   "Create a new algebraic data type named NAME. (Coalton top-level operator.)")
 
+(define-coalton-editor-macro coalton:define-alias (name &body definition)
+  "Create a new alias named NAME. (Coalton top-level operator.)")
```

Comment:
This is specifically a type alias, not an alias for any name.

Maybe we should go as far as rename the operator

    define-type-alias

Reply:
resolved in [57366b7](https://github.com/coalton-lang/coalton/pull/1294/commits/57366b7f79147a4d910d218292688914c7a876a8)